packages/react-docs/patternfly-a11y.config.js

```diff
@@ -16,5 +16,5 @@ module.exports = {
   urls: Object.keys(fullscreenRoutes),
   ignoreRules: 'color-contrast,page-has-heading-one,scrollable-region-focusable,bypass',
   ignoreIncomplete: true,
-  skip: /^\/charts\//
+  skip: /(^\/charts\/)|(^\/components\/table\/react\/composable-tree-table)|(^\/components\/table\/react\/tree-table)/
```

Comment:
you could make this more concise if you like
`skip: /^\/charts\/|tree-table$/`
maybe also add a comment why they are skipped?